web/src/components/Header.tsx

```diff
@@ -57,12 +57,6 @@ const Row = styled.div`
     min-height: ${dimensions.headerHeightSmall}px;
     overflow-x: auto;
     padding: 8px 0;
-    box-shadow: 0 2px 5px -3px rgb(0 0 0 / 20%);
-
-    :first-child {
-      box-shadow: 0 2px 5px -3px rgb(0 0 0 / 12%);
-      padding: 0 4px;
-    }
```

Comment:
Does anybody have an idea why this was needed? And why this was not a problem before and only started leading to this weird shadow now?

Reply:
the answer to that question can be found here:
https://github.com/digitalfabrik/integreat-app/blob/821b2b0fb666f6f90a602fc869c659f8466377f5/web/package.json#L58 @steffenkleinle 
But no idea why sarah added this shadows tbh
At least update of styled components library made this style working